### ISSUE TITLE
 SyntaxError: Cannot use import statement outside a module with tailwindcss-react-aria-components

### ISSUE THREAD

**Issue by jared-christensen** [adobe/react-spectrum#5221]
### Provide a general summary of the issue here

The plugin tailwindcss-react-aria-components seems to be using ESM imports, causing a syntax error when used in a Tailwind config which generally uses CommonJS

### 🤔 Expected Behavior?

The plugin should be compatible with the Tailwind config without throwing ESM-related errors.

### 😯 Current Behavior

Console throws an error: `SyntaxError: Cannot use import statement outside a module`

### 💁 Possible Solution

Could the plugin be refactored to support CommonJS imports or offer a CommonJS version?

### 🔦 Context

I'm trying to integrate this plugin into my project's Tailwind configuration. This issue prevents me from doing so.

### 🖥️ Steps to Reproduce

1. Install tailwindcss-react-aria-components
2. Add require('tailwindcss-react-aria-components') to the plugins array in tailwind.config.js
3. Run the development server

### Version

tailwindcss-react-aria-components: 1.0.0-beta.1

### What browsers are you seeing the problem on?

Chrome

### If other, please specify.

Im working in NX and Next.js

### What operating system are you using?

macOS

### 🧢 Your Company/Team

DHI Group Inc

**Comment by jared-christensen** [comment]
I think this is the import that is causing me trouble. https://github.com/adobe/react-spectrum/blob/main/packages/tailwindcss-react-aria-components/src/index.js#L1

**Comment by jared-christensen** [comment]
I take it all back, I updated to the newest version of tailwind and it all works fine now. https://tailwindcss.com/blog/tailwindcss-v3-3#esm-and-type-script-support
